src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -284,15 +284,15 @@ namespace winrt::TerminalApp::implementation
     void TerminalPage::_ShowCloseWarningDialog()
     {
         auto title = RS_(L"CloseWindowWarningTitle");
-        auto primaryButtonText = RS_(L"CloseAll");
-        auto secondaryButtonText = RS_(L"Cancel");
+        auto closeButtonText = RS_(L"CloseAll");
+        auto primaryButtonText = RS_(L"Cancel");
 
         WUX::Controls::ContentDialog dialog;
         dialog.Title(winrt::box_value(title));
 
+        dialog.CloseButtonText(closeButtonText);
```

Comment:
wow, we were using the wrong button!

Reply:
I am actually somewhat concerned about this. :)
What if you press _escape_ while the dialog is open?

Reply:
> Use the CloseButtonText and property to set the display text for the safe, non-destructive button. The dialog's close button will also be invoked when the user performs a Cancel action, like pressing the ESC key or pressing the system back button. Use the PrimaryButtonText and SecondaryButtonText properties to display responses to the main question or action posed by the dialog.

big bad news

Reply:
the close button is for closing the dialog, not for closing the application. :smile:

Reply:
Should I leave it as the primary button but style it with the active coloring? (same as how we styled the close button)